polkadot/runtime/westend/src/lib.rs

```diff
@@ -162,7 +162,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("westend"),
 	impl_name: create_runtime_str!("parity-westend"),
 	authoring_version: 2,
-	spec_version: 1_015_000,
+	spec_version: 1_016_002,
```

Comment:
this change seems unrelated

Reply:
an artifact of testing the runtime upgrade